plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipment.java

```diff
@@ -87,6 +89,12 @@ else if (attribute.startsWith("equipment.leggings") || attribute.startsWith("equ
                 ItemStack leggings = equipment.getLeggings();
                 return new ItemTag(leggings != null ? leggings : new ItemStack(Material.AIR));
             }
+            else if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20) && attribute.startsWith("equipment.body")) {
+                BukkitImplDeprecations.entityEquipmentSubtags.warn(attribute.context);
+                attribute.fulfill(1);
+                ItemStack body = equipment.getItem(EquipmentSlot.BODY);
+                return new ItemTag(body);
+            }
```

Comment:
Shouldn't add new deprecated tags, new slots like `body` will only be used in new scripts either way, which would be using `equipment_map`.